package.json

```diff
@@ -23,7 +23,7 @@
     "eslint-plugin-sonarjs": "0.5.0",
     "if-env": "^1.0.4",
     "import-sort-style-eslint": "^6.0.0",
-    "lerna": "^4.0.0",
+    "lerna": "^2.11.0",
```

Comment:
just fyi, as far as I understand:
- this change has no influence for building theia + che-theia assembly
- this change has no influence for building CRW assembly
- this change has influence only for building che-theia as a project